src/emulator/gxm/src/gxm.cpp

```diff
@@ -17,7 +18,7 @@
 
 static std::string load_shader(const char *hash, const char *extension, const char *base_path) {
     std::ostringstream path;
-    path << base_path << "shaders/" << hash << "." << extension;
+    path << base_path << "/" << hash << "." << extension;
```

Comment:
I think this change needs to be undone, as `base_path << "shaders/"` is the path to the current shaders.